lib/ClangImporter/ImportDecl.cpp

```diff
@@ -345,6 +345,23 @@ static bool isNSDictionaryMethod(const clang::ObjCMethodDecl *MD,
   return true;
 }
 
+static void forEachDistinctName(
```

Comment:
Hm. It's each distinct name and context to import the declaration, at most one per Swift version. I'm happy to put that in a doc comment, but do you have a better name for the function itself?